### ISSUE TITLE
Cursor positioning still not right in 2.0.5

### ISSUE THREAD

**Issue by borekb** [careteditor/caret#271]
Previously: https://github.com/careteditor/caret/issues/264

I still see some issue, for example, the caret hiding a dot:

![image](https://cloud.githubusercontent.com/assets/101152/24446934/9b7dc424-146f-11e7-8cac-34b6a47b4610.png)

![image](https://cloud.githubusercontent.com/assets/101152/24446943/a041f43a-146f-11e7-941f-062a59c9719a.png)

**Comment by astoilkov** [comment]
Can you share the line of text with the problem and the font you are using? Thanks in advance.

**Comment by borekb** [comment]
```
The style is adapted from from [sindresorhus/github-markdown-css](https://github.com/sindresorhus/github-markdown-css), version [`322382b`](https://github.com/sindresorhus/github-markdown-css/blob/322382b0504213e608cba0449c95b3fe98ca8ec9/github-markdown.css) which is also copied to `github-markdown.css` in this repo. You can compare `github-markdown.css` to `New GitHub.css` to see what changes have been done.
```

**Comment by astoilkov** [comment]
Can you share the font family and line width from your `Preferences.md`?

**Comment by astoilkov** [comment]
And one last question :) - does the last line in the wrapped line contain the following text: "to `New GitHub.css` to see what changes have been done."

**Comment by borekb** [comment]
Re: Preferences:

```yaml
font: Segoe UI
fontSize: 21
lineHeight: 1.9
lineWidth: 72
```

Re: line wrapping: it doesn't make a difference:

![image](https://cloud.githubusercontent.com/assets/101152/24452238/8cf02150-1482-11e7-9cd6-1c1dc124c2ef.png)


![image](https://cloud.githubusercontent.com/assets/101152/24452227/7f69c0b8-1482-11e7-9fb9-84e055290d8e.png)

**Comment by astoilkov** [comment]
This issue is now fixed in the Caret 2.0.6 release. The problem wasn't that the cursor isn't at the correct possible but the caret was too wide. We made it thinner so it doesn't hide characters.

Closing this issue. Tell us if there is another problem.
